drivers/wdm/audio/backpln/portcls/undoc.cpp

```diff
@@ -722,6 +605,7 @@ DumpFilterDescriptor(
         }
         DPRINT("------ End of Nodes Connections----------------\n");
     }
+
```

Comment:
Skip that cosmetical change for the sake of minimizing.

Reply:
Fixed.